tests/ert/unit_tests/scheduler/test_slurm_driver.py

```diff
@@ -458,3 +460,37 @@ async def test_slurm_uses_sacct(
 
     # Make sure sacct was tried:
     assert "scontrol failed, trying sacct" in caplog.text
+
+
+async def test_slurm_timeout(tmp_path, caplog, pytestconfig):
+    os.chdir(tmp_path)
+    caplog.set_level(logging.INFO)
+
+    if pytestconfig.getoption("slurm"):
+        cmds = ["sleep 300"]
```

Comment:
(I suppose plural form of `cmd` is misleading here, as a list here is only one command)

Reply:
Indeed, will fix.